src/azure-cli/azure/cli/command_modules/acr/manifest.py

```diff
@@ -0,0 +1,454 @@
+        json_payload=None,
+        permission=RepoAccessTokenPermission.METADATA_READ.value,
+        repository=None,
+        image=manifest_spec,
+        tenant_suffix=tenant_suffix,
+        username=username,
+        password=password)
+
+
+def acr_manifest_metadata_update(cmd,
+                                 registry_name=None,
+                                 manifest_spec=None,
+                                 manifest_id=None,
+                                 tenant_suffix=None,
+                                 username=None,
+                                 password=None,
+                                 delete_enabled=None,
+                                 list_enabled=None,
+                                 read_enabled=None,
+                                 write_enabled=None):
+    if (manifest_id and manifest_spec) or (not manifest_id and not (registry_name and manifest_spec)):
+        raise InvalidArgumentValueError(BAD_ARGS_ERROR_MANIFEST)
+
+    if manifest_id:
+        registry_name, repository, tag, manifest = _parse_fqdn(cmd, manifest_id[0])
+        manifest_spec = repository + ':' + tag if tag else repository + '@' + manifest
+
+    json_payload = {}
+
+    if delete_enabled is not None:
+        json_payload.update({
+            'deleteEnabled': delete_enabled
+        })
+    if list_enabled is not None:
+        json_payload.update({
+            'listEnabled': list_enabled
+        })
+    if read_enabled is not None:
+        json_payload.update({
+            'readEnabled': read_enabled
+        })
+    if write_enabled is not None:
+        json_payload.update({
+            'writeEnabled': write_enabled
+        })
+
+    permission = RepoAccessTokenPermission.META_WRITE_META_READ.value if json_payload \
+        else RepoAccessTokenPermission.METADATA_READ.value
+    return _acr_repository_attributes_helper(
+        cmd=cmd,
+        registry_name=registry_name,
+        http_method='patch' if json_payload else 'get',
+        json_payload=json_payload,
+        permission=permission,
+        repository=None,
+        image=manifest_spec,
+        tenant_suffix=tenant_suffix,
+        username=username,
+        password=password)
+
+
+def acr_manifest_delete(cmd,
+                        registry_name=None,
+                        manifest_spec=None,
+                        manifest_id=None,
+                        tenant_suffix=None,
+                        username=None,
+                        password=None,
+                        yes=False):
+    if (manifest_id and manifest_spec) or (not manifest_id and not (registry_name and manifest_spec)):
+        raise InvalidArgumentValueError(BAD_ARGS_ERROR_MANIFEST)
+
+    if manifest_id:
+        registry_name, repository, tag, manifest = _parse_fqdn(cmd, manifest_id[0])
+
+    else:
+        repository, tag, manifest = _parse_image_name(manifest_spec, allow_digest=True)
+
+    if not manifest:
+        image = repository + ':' + tag
```

Comment:
You use it several times. Might as well create a constant call repoTagLimiter or something like that